package.json

```diff
@@ -6,13 +6,14 @@
   "scripts": {
     "analyze:size": "webpack --config webpack.prod.js --profile --json=bundle-stats.json && webpack-bundle-analyzer bundle-stats.json dist/",
     "build": "webpack --config webpack.prod.js",
-    "start": "webpack serve --open --config webpack.dev.js",
     "lint": "npm run lint:check",
     "lint:check": "npm-run-all --sequential --continue-on-error lint:check:*",
     "lint:check:css": "stylelint src/**/*.css",
-    "lint:check:js": "eslint src webpack.*.js --max-warnings 0",
+    "lint:check:js": "eslint jest.config.ts src webpack.*.js --max-warnings 0",
```

Comment:
Don't we want to lint the test files too?

Reply:
Sorry. Addressed in e4f90d427e4c9158a8e7ae6fdd9ae9a81afad91f